BuilderBuilder/UI/SettingsControl.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Avalonia.Controls;
+using Avalonia.Interactivity;
+using Avalonia.Markup.Declarative;
+using AvaloniaExtensions;
+
+namespace BuilderBuilder.UI;
+
+internal class SettingsControl : CanvasComponentBase {
+  private const int LABEL_WIDTH = 110;
+
+  private Settings Settings => GetSettings<Settings>();
+
+  private TextBox _tbNamespace = null!;
+  private RadioButton _rbSpaces = null!, _rbTabs = null!;
+  private TextBox _tbNumberOfSpaces = null!;
+  private RadioButton _rbEgyptianBraces = null!, _rbNormalBraces = null!;
+
+  protected override void InitializeControls() {
+    _tbNamespace = AddTextBox().TopLeftInPanel();
+    _rbSpaces = AddRadio("indent", "Spaces").Below();
+    _rbTabs = AddRadio("indent", "Tabs").Below();
+    _rbNormalBraces = AddRadio("braces", "Normal (Allman)").Below();
+    _rbEgyptianBraces = AddRadio("braces", "Egyptian (K&R)").Below();
+
+    InsertLabelLeftOf("Namespace:", _tbNamespace, LABEL_WIDTH);
+    InsertLabelLeftOf("Indent size:", _rbSpaces, LABEL_WIDTH);
+    InsertLabelLeftOf("Brace style:", _rbNormalBraces, LABEL_WIDTH);
+    _rbTabs.XAlignLeft(_rbSpaces);
+    _rbEgyptianBraces.XAlignLeft(_rbNormalBraces);
+
+    _tbNumberOfSpaces = AddTextBox().OnTextChanged(_ => _rbSpaces.IsChecked = true).RightOf(_rbSpaces);
```

Comment:
Unfortunately, this sometimes selects the spaces radio when tabs should be selected.